lib/features/children/overview/widgets/family_available_page.dart

```diff
@@ -14,15 +15,26 @@ class FamilyAvailablePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final currentUserFirstName = context.read<AuthCubit>().state.user.firstName;
+    // The Givt user profile is first in the list
+    final sortedAdultProfiles = profiles.where((p) => p.isAdult).toList()
+      ..sort((a, b) {
+        final compareNames = a.firstName.compareTo(b.firstName);
+        return a.firstName == currentUserFirstName
+            ? -1
+            : b.firstName == currentUserFirstName
+                ? 1
+                : compareNames;
+      });
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        ParentOverviewWidget(
-          profiles: profiles.where((p) => p.type == 'Parent').toList(),
+        ProfilesOverviewWidget(
+          profiles: sortedAdultProfiles,
         ),
         const SizedBox(height: 20),
-        ChildrenOverviewWidget(
-          profiles: profiles.where((p) => p.type == 'Child').toList(),
+        ProfilesOverviewWidget(
+          profiles: profiles.where((p) => p.isChild).toList(),
```

Comment:
same here: what about _children_ getter (with filtering inside) in the cubit?

Reply:
ah since the widget was getting the list of profile instead of state i overlooked those, oops